lnt/server/ui/views.py

```diff
@@ -960,30 +974,13 @@ def v4_tableau():
 
 @v4_route("/graph")
 def v4_graph():
-
     session = request.session
     ts = request.get_testsuite()
-    switch_min_mean_local = False
```

Comment:
Honestly, I don't really understand where there was so much complexity around keeping a local version of the "switch to min aggregation". In this patch I'm just treating it as a normal parameter of the graph (like other parameters) and that seems to work fine. I hope I didn't miss something subtler.